SECURITY.md

```diff
@@ -0,0 +1,93 @@
+- You are looking for help applying security updates.
+
+### Security Vulnerability Response
+Each report will be reviewed and receipt acknowledged within 3 business days. This will set off the security review process detailed below.
+
+Any vulnerability information shared with the security team stays within the Notary Project and will not be shared with others unless it is necessary to fix the issue. Information is shared only on a need to know basis.
+
+We ask that vulnerability reporter(s) act in good faith by not disclosing the issue to others. And we strive to act in good faith by acting swiftly, and by justly crediting the vulnerability reporter(s) in writing.
+
+As the security issue moves through triage, identification, and release the reporter of the security vulnerability will be notified. Additional questions about the vulnerability may also be asked of the reporter.
+
+### Public Disclosure
+A public disclosure of security vulnerabilities is released alongside release updates or details that fix the vulnerability. We try to fully disclose vulnerabilities once a mitigation strategy is available. Our goal is to perform a release and public disclosure quickly and in a timetable that works well for users. For example, a release may be ready on a Friday but for the sake of users may be delayed to a Monday.
+
+CVEs will be assigned to vulnerabilities. Due to the process and time it takes to obtain a CVE ID, disclosures will happen first. Once the disclosure is public the process will begin to obtain a CVE ID. Once the ID has been assigned the disclosure will be updated.
+
+If the vulnerability reporter would like their name and details shared as part of the disclosure process we are happy to. We will ask permission and for the way the reporter would like to be identified. We appreciate vulnerability reports and would like to credit reporters if they would like the credit.
+
+Vulnerability disclosures are published in the Security Advisories sections in each repository. The disclosures will contain an overview, details about the vulnerability, a fix for the vulnerability that will typically be an update, and optionally a workaround if one is available.
+
+Disclosures will be published on the same day as a release fixing the vulnerability after the release is published.
+
+Here are the links to security advisories for each repository:
+
+- [Notation CLI Security Advisories](https://github.com/notaryproject/notation/security/advisories)
+- [Notation-Go Library Security Advisories](https://github.com/notaryproject/notation-go/security/advisories)
+- [Notation-Core-Go Library Security Advisories](https://github.com/notaryproject/notation-go/security/advisories)
+- [Notation Spcecification Security Advisories](https://github.com/notaryproject/notaryproject/security/advisories)
+
+## Security Team Membership
+The security team is made up of a subset of the Notation project maintainers who are willing and able to respond to vulnerability reports.
```

Comment:
Are the sub-repo maintainers by default the members of the security team? Or do we need to nominate people to security team per sub-repo?

Reply:
The sub-project maintainers are contacted by default. Subset can acknowledge and add additional members if relevant.

Reply:
Updated the text to clarify.